packages/Webkul/Admin/src/DataGrids/ProductDataGrid.php

```diff
@@ -166,6 +167,9 @@ public function addColumns()
             'searchable' => true,
             'sortable'   => true,
             'filterable' => true,
+            'closure'    => function ($row) {
+                return "<a href='".asset($row->url_key)."' target='_blank'>" . $row->product_name . "</a>";
```

Comment:
Use space before and after with binary operator

Reply:
there should be a front_url in the .env file and the same should be used here if the frontend app is different than the default such as SPAs.